Unity-Technologies/Graphics
# HDRP/Fix package version showing package after the last "verified" package

After enquiering on FB issue on package version shown in wizard not the same as the one shown in package manager windows, it seams that this is expected behaviour of package lifecycle v2.
Only remaining :
- a typo (fixed in this PR)
- a design decision to make: align Wizard with Package Manager or remains with old behaviour for package last version discoverability


**UPDATE:**

Removed last package version check from wizard and add a button for shortchut check this in Package Manager. So now we have alignment.
![image](https://user-images.githubusercontent.com/40034005/100629909-23f78880-332a-11eb-92d4-f3517912caf4.png)


### Tests done
Manually tested that:
- the version of current HDRP package appears. Tested with local package only. "(local)" should only appears at end for local packages.
- the retrieval of config package local installation still work. Changes introduced a regression on this due to asynchrone call conflict. It is handled with a stack so we should not bother on order but previous code only did it if nothing was in the stack. I dunno the reason. In the test, no local package was installed so it found it and display the installation button after the fix.
- the **Check update** button open the Package Manager window on the HDRP package. Also tested with the window already open. Not tested with the window opened on another package but this is package manager API so I assume it was tested on their side.
